pkgs/applications/graphics/imv/default.nix

```
@@ -90,6 +90,11 @@ stdenv.mkDerivation rec {
   ] ++ windowSystems."${withWindowSystem}"
     ++ builtins.map (b: backends."${b}") withBackends;
 
+  postInstall = ''
+    mkdir -p $out/share/applications
+    cp ../files/imv.desktop $out/share/applications/
```

```suggestion
    # fix the executable path and install the desktop item
    substituteInPlace ../files/imv.desktop --replace "imv %F" "$out/bin/imv %F"
    install -Dm644 ../files/imv.desktop $out/share/applications/
```
It's better to use a store path to make sure the result is pure.